packages/@react-spectrum/dropzone/stories/DropZone.stories.tsx

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright 2023 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {action} from '@storybook/addon-actions';
+import {Content} from '@react-spectrum/view';
+import {Draggable} from '@react-aria/dnd/stories/dnd.stories';
+import {DropZone} from '../';
+import {FileDropItem, TextDropItem} from 'react-aria';
+import {FileTrigger, Text} from 'react-aria-components';
+import {Heading} from '@react-spectrum/text';
+import {IllustratedMessage} from '@react-spectrum/illustratedmessage';
+import {Link} from '@react-spectrum/link';
+import {Meta} from '@storybook/react';
+import React, {useState} from 'react';
+import {SpectrumDropZoneProps} from '../src/DropZone';
+import Upload from '@spectrum-icons/illustrations/Upload';
+
+type StoryArgs = SpectrumDropZoneProps;
+
+const meta: Meta<StoryArgs> = {
+  title: 'DropZone',
+  component: DropZone
+};
+
+export default meta;
+
+export const withDraggable = {
+  render: (args) => (
+    <DropZoneWithDraggable {...args} />
+  )
+};
+
+export const withLink = {
+  render: (args) => (
+    <DropZoneWithLink {...args} />
+  )
+};
+
+export const customBannerMessage = {
+  render: (args) => (
+    <Example 
+      {...args}
+      bannerMessage="This is a custom message" />
+  )
+};
+
+export const acceptsMultiple = {
+  render: (args) => (
+    <Example {...args} />
+  )
+};
+
+function Example(props) {
+  let [isFilled, setIsFilled] = useState(false);
+  let [filledSrc, setFilledSrc] = useState(null);
+  
+  return (
+    <DropZone 
+      {...props}
+      isFilled={isFilled}
+      onDrop={async (e) => { 
+        let items = e.items.filter((item) => item.kind === 'file') as FileDropItem[];
+        if (items.length > 0) {
+          const urls = await Promise.all(items.map(async (item) => URL.createObjectURL(await item.getFile())));
+          const stringUrls = urls.map((url) => url.toString());
+          setFilledSrc(stringUrls);
+          setIsFilled(true);
+        }
+      }}
+      onDropEnter={action('onDropEnter')}
+      onDropExit={action('onDropExit')} 
+      onPaste={async (e) => { 
```

Comment:
Maybe a question for the team, but should we just have pasting trigger a drop automatically? Feels strange to force users to handle both.

Reply:
Looks like we do. In that case, this story can be updated to just do `action('onPaste')`

Reply:
oops my bad! i'll change it to `action('onPaste')`